pkg/exchange/max/stream.go

```diff
@@ -119,14 +130,16 @@ func (s *Stream) handleConnect() {
 			}
 		}
 
+		log.Debugf("user data websocket filters: %v", filters)
```

Comment:
Do you need a sanity check for filters? to avoid zero subscription

Reply:
It's ok, because empty filters = default filters